src/stdune/fpath.mli

```diff
@@ -6,3 +6,10 @@ type mkdir_p =
   | Created  (** The directory was created. *)
 
 val mkdir_p : ?perms:int -> string -> mkdir_p
+
+type follow_symlink_error =
+  | Not_a_symlink
+  | Max_depth_exceeded
+  | Unix_error of exn
```

Comment:
In other places we use `Unix.error` instead of `exn`, which makes matching on it safer. Should we do the same here?
(I know the exception has a bit extra info, so not sure if you did this deliberately)

Reply:
Changed it to `Unix.error`. I forgot we used this elsewhere.